app/Services/APIServices.php

```diff
@@ -463,7 +463,7 @@ public function getAllContracts($request)
             ];
         }
 
-        $params['body']['query']['filtered']['filter']['and']['filters'] = $filter;
+        $params['body']['query']["bool"]['filter'] = $filter;
```

Comment:
This would look more consistent if we would use single quotes everywhere.

Reply:
I've just seen you comment. Changed to single quotes. Thank you!